src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,96 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int PRIME_NUMBER = 17;
```

Comment:
I don't think you need PRIME_NUMBER constant here, as it more connected to hashCode that hashMap. You can make it  local variable in hashCode() method

Reply:
removed it at all